test/shared/src/test/scala/zio/test/mock/LiveSpec.scala

```diff
@@ -0,0 +1,25 @@
+package zio.test.mock
+
+import java.util.concurrent.TimeUnit
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+import zio.clock
+import zio.DefaultRuntime
+import zio.test.TestUtils.label
+
+object LiveSpec extends DefaultRuntime {
+
+  def run(implicit ec: ExecutionContext): List[Future[(Boolean, String)]] = List(
+    label(liveCanAccessRealEnvironment, "live can access real environment")
+  )
+
+  def liveCanAccessRealEnvironment: Future[Boolean] =
+    unsafeRunToFuture {
+      val io = for {
+        mock <- clock.currentTime(TimeUnit.MILLISECONDS)
+        live <- Live.live(clock.currentTime(TimeUnit.MILLISECONDS))
+      } yield mock == 0 && live != 0
```

Comment:
Really love how this looks! We should consider re-exporting `live` as a method inside the `mock` package object, so a `import zio.test.mock._` brings it into scope.

Reply:
Yes, it is really nice. I re-exported it in the package object in the latest version.